### PR TITLE
Add Image component page to docs

### PR DESCRIPTION
**Pull request by mcmonojit**
**Added image component page to documentation**

- responsive image
- circular image

### CODE REVIEW

File: docs/image/image.html [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>CoolWaterUI</title>
+    <link rel="stylesheet" href="image.css">
+</head>
+<body>
+    <div class="sidebar-and-main-container">
+        <aside class="sidebar">
+
+            <div class="docs-navbrand-container"><a href="/"><div class="navbar-brand">CoolWaterUI</div></a></div>
```
Comment (chanchal16) [inline docs/image/image.html:16]: Here there's no need to put your brand name inside div <a> tag is enough. You can remove that div if there's no valid reason for it.

**Review by chanchal16 — APPROVED** [review]
Left a suggestion. Also, your HTML code has some extra blank spaces you can remove.

- Your code looks clean.
- meaningful and readable classnames 
- The UI looks good.

**Comment by mcmonojit** [comment]
Thanks @chanchal16, for the review. The div has been used to easily apply the styling on the brand name and I do not know an alternative method to implement the same without the div. If I can find another method I will surely implement it.

File: docs/image/image.html [hunk 2]
```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (bharati-21) [inline docs/image/image.html:6]: You can add meta author and desc tags for increasing the accessibility

File: docs/image/image.html [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+                        class="responsive-img"
+                    />
+                </div>
+
+                <p>The following code snippet contains the code for the above components.</p>
+
+                <div class="component-container code-snip-width">
+                    <iframe
+                        src="https://carbon.now.sh/embed/5wIEo6mGjLnZBbtvUcyF"
+                        style="width: 571px; height: 279px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (bharati-21) [inline docs/image/image.html:96]: Kindly remove inline styles. You can add a class and the related styles on the class.

File: docs/image/image.html [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+                                class="circular-img"/>
+                        </div>
+                </div>
+
+                <p>The following code snippet contains the code for the above components.</p>
+
+                <div class="component-container code-snip-width">
+                    <iframe
+                        src="https://carbon.now.sh/embed/LtUzio1QAlO9xUz2Kasq"
+                        style="width: 563px; height: 335px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (bharati-21) [inline docs/image/image.html:124]: Same suggestion here as well.

**Review by bharati-21 — APPROVED** [review]
You can look at the formating and spaces between lines of code a little. Maybe an extension to do this. Your folder structure is wonderful! Adding HTML and the associated CSS within the same folder is nice!

Apart from this, your code looks good to go.

**Comment by mcmonojit** [comment]
Thanks @bharati-21, for  the review. I have implemented the recommended changes in [9cbf67e](https://github.com/mcmonojit/CoolWaterUI/pull/5/commits/9cbf67e4fbeb6f9dc4fb97dee4f39e179a146cef).

**Pull request merged**